heareval/predictions/task_predictions.py

```diff
@@ -15,10 +15,13 @@
 """
```

Comment:
This file is too long, but I don't want to break it into ten files just for the sake of it. I wonder what is the most logical way to break it down.

Reply:
Ya it's a bit long -- but not the worst. I was going to say we could move the prediction models to there own module -- but maybe it is kind of nice to have them in the same file. I don't think it is a huge issue. I'm kind of into the idea of moving the event creation from frames functions into the scoring modules.